public/myScript/orders.js

```diff
@@ -0,0 +1,158 @@
+
+
+var dynamicTable = (function() {
+
+    var _tableId, _table,
+        _fields, _headers,
+        _defaultText;
+
+    /** Builds the row with columns from the specified names.
+     *  If the item parameter is specified, the memebers of the names array will be used as property names of the item; otherwise they will be directly parsed as text.
+     */
+
+    function format(item){
+            var tagcolor = '<tr>';
+            switch (item){
+                case 4:tagcolor = '<tr bgcolor = " #00FF7F">';break;
```

Comment:
Please, use CSS for styling. Here you can define a class for this `tr` and then this class styling can be defined in CSS.